src/__tests__/unit/calculate.test.js

```diff
@@ -0,0 +1,26 @@
+import calculate from '../../logic/calculate';
+
+describe('Validate calculate functionality', () => {
+  const num = { total: null, next: null, operation: null };
+  test('multiplication', () => {
+    let result = calculate(num, '10');
+    result = calculate(result, 'x');
+    result = calculate(result, '5');
+    result = calculate(result, '=');
+    expect(result.total).toBe('50');
+  });
+  test('plus', () => {
+    let result = calculate(num, '10');
+    result = calculate(result, '+');
+    result = calculate(result, '5');
+    result = calculate(result, '=');
+    expect(result.total).toBe('15');
+  });
+  test('divide', () => {
+    let result = calculate(num, '10');
+    result = calculate(result, '÷');
+    result = calculate(result, '5');
+    result = calculate(result, '=');
+    expect(result.total).toBe('2');
```

Comment:
- [ ] Please write tests for other operators.

Reply:
Thanks for this awesome awake call. We initially thought a few would do. We will implement all the other pending tests.

> Cheers and HappyCoding 👨🏽‍💻

Reply:
> Done ✅